shared/locales/langpack/desc.en-US.properties

```diff
@@ -0,0 +1,34 @@
+# This file provides localization strings for the Language Pack.
```

Comment:
Maybe use the standard `LOCALIZATION_NOTE` header here and before each group of strings in this file, in case tools rely on it to parse localization comments?